src/components/ui4/BlogCards.tsx

```diff
@@ -13,6 +13,34 @@ export interface Props {
   countryCode?: string
 }
 
+/**
+ * Filters a list of blog posts based on the provided country code.
+ *
+ * @param cards - An array of `BlogPostGeo` objects representing the blog posts to filter.
+ * @param countryCode - An optional string representing the country code to filter by. If `null`, an empty array is returned.
+ * @returns An array of `BlogPostGeo` objects that match the provided country code.
+ */
+export const filterBlogCards = (cards: BlogPostGeo[], countryCode?: string) => {
+  if (countryCode == null) return []
+
+  return cards.filter((blog: BlogPostGeo) => {
+    const { countryCodes: includeCountryCodes = [], excludeCountryCodes = [] } = blog
+
+    const isCountryInclude =
+      includeCountryCodes.length === 0 || (includeCountryCodes as string[]).some(includeCountryCode => includeCountryCode.toUpperCase() === countryCode)
+    const isCountryExclude =
```

Comment:
isCountryExcluded and isCountryExcluded would be better grammar

Reply:
I prioritize consistency and clarity in variable naming over adherence to standard grammar rules, focusing on the orderly organization of words and maintaining verb tenses relevant to the present context, such as using "Include" or "Exclude." I avoid prefixes like "has" or "was" that indicate tense, as they can lead to a disorganized naming structure.

This approach enhances readability, particularly for non-native English speakers, by maintaining a uniform format. For instance, despite "info" being both singular and plural in correct English, I prefer using "infos" for collections to ensure the meaning is clear, even though it is grammatically incorrect no matter the context.

Reply:
Not sure how the argument above equates to `isCountryExclude` being a better option. It doesn't really provide consistency to anything else. Still confused by the decision but I'm not married to my comment.  Since it's just a local var I'll let it pass. If this were an API I would push harder.

Reply:
I'm sure this'll come up again, I might just have really outdated training